templates/_helpers.tpl

```diff
@@ -73,3 +73,67 @@ port: 5672
 username: {{ .Values.rabbitmq.auth.username }}
 password: ${RABBITMQ_PASSWORD}
 {{- end }}
+
+
+{{- define "geoserver.common.env.variables" -}}
+- name: EUREKA_SERVER_URL
+  value: http://{{ include "geoserver.fullname" . }}-discovery:8761/eureka
+- name: BACKEND_CATALOG
+  value: "false"
```

Comment:
I prefer to put the static env in the values.yaml to be able to override them :-)